service/build.gradle

```diff
@@ -42,9 +42,9 @@ dependencies {
     // at commit time, azure-resourcemanager-monitor v2.20.0 is the latest but starting with v2.19.0
     // attributes of com.azure.resourcemanager.monitor.fluent.models.DataCollectionRuleResourceInner
     // are missing making creation of DataCollectionRules impossible via java client lib
-    implementation group: 'com.azure.resourcemanager', name: 'azure-resourcemanager-monitor', version: '2.27.0'
+    implementation group: 'com.azure.resourcemanager', name: 'azure-resourcemanager-monitor', version: '2.28.0'
```

Comment:
Same as this PR, needs to be here for the test to work. https://github.com/DataBiosphere/terra-landing-zone-service/pull/163